app/lib/project/MetapolatorProject.js

```diff
@@ -509,8 +509,54 @@ define([
         return this._controller;
     };
 
-    _p.import = function(masterName, sourceUFODir, glyphs) {
-        var importer = new ImportController( this._log, this,
+    _p.importZippedUFOInstances = function(filename, blob) {
+        var mem_io = new InMemory()
+          , imported_instances = Array()
+          ;
+
+        zipUtil.unpack(false, blob, mem_io, "");
+
+        var dirs = mem_io.readDir(false, "/")
+          , baseDir = dirs[0]
+          , names = mem_io.readDir(false, baseDir)
+          , n, l
+          ;
+
+        console.log("dirs:", dirs);
+
+        for (n=0, l=names.length; n<l; n++){
+            var name = names[n]
+              , UFOZip = baseDir + name //This may be wrong in some cases. We need a more robust implementation.
```

Comment:
We discussed that in issue #344 and Dave suggested we start with something simple. So right now we only support the scheme described here:

https://plus.google.com/114560892745622996493/posts/he2KdmJ7iDv